i18n/languages/en.json

```diff
@@ -358,8 +358,8 @@
     "Settings.Sections.Global.WaitLongForRepair.Description": "Should the max repair time be 60 seconds instead of 15?",
     "Settings.Sections.Global.WaitLongForRepair": "Enable long repair",
     "Settings.Sections.Global": "Admin Options",
-    "Settings.Sections.GUI.BigPopup.Description": "Should the popup for new visual GUI be bigger?",
-    "Settings.Sections.GUI.BigPopup": "GUI big popup",
+    "Settings.Sections.GUI.PopupScale.Description": "Allows to scale the popup for new visual GUI",
```

Comment:
Can you ensure that these language string are alphabetical within the file?

Reply:
Done